projects/plugins/jetpack/_inc/lib/admin-pages/class.jetpack-admin-page.php

```diff
@@ -174,11 +203,11 @@ function is_rest_api_enabled() {
 	 *
 	 * @since 4.4.0
 	 *
-	 * @param $page
+	 * @param Page $page string Pages related to paid modules.
```

Comment:
Looks like this function is used with the `current_screen` action, which describes the parameter as
```suggestion
	 * @param WP_Screen $page Current WP_Screen object.
```